src/commcare_cloud/commands/ansible/helpers.py

```diff
@@ -84,11 +84,7 @@ def get_default_ssh_options(environment, aws_ssm_target=None):
             '--document-name AWS-StartSSHSession '
             '--parameters portNumber=%p'
         )
-        assert '"' not in sso_command, """
-            Use {shlex_quote(sso_command)} and remove quotes (") in the
-            ProxyCommand below if it becomes necessary to put quotation
-            marks in sso_command. Otherwise it is more readable as is."""
-        default_ssh_options.append(('ProxyCommand', f'sh -c "{sso_command}"'))
+        default_ssh_options.append(('ProxyCommand', sso_command))
```

Comment:
The `sso_command` arg should be quoted.

Reply:
The previous `sh -c ...` command was not quoted here. The option gets quoted later by `subprocess.call()`

https://github.com/dimagi/commcare-cloud/blob/f44bb73689131adaf04592c258e15128d7118aab/src/commcare_cloud/commands/inventory_lookup/inventory_lookup.py#L89-L93

The `print_command()` output looks like this:

`-o 'ProxyCommand=aws ssm start-session --target ... --document-name AWS-StartSSHSession --parameters portNumber=%p'`

Reply:
Ah, brilliant!